news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown

```diff
@@ -0,0 +1,266 @@
+| Actions | All the call-to-actions, buttons or links that you find on the back office, and that are quite generic (“Save”, “Add”, “Delete”, etc.). Again, if it applies to one page and one functional domain only, then this is not the domain to use. |
+| Notifications | All the warning, error or success messages that can appear in the back office.They must be general notifications applying to any part of the software (e.g. “Settings updated.”).<br/> In case you have a specific notification (“Friendly URLs are currently disabled”), then it should fall under the notification subdomain for its functional domain (here *Admin.Catalog.Notification*). |
+| Navigation | The structure shared by all pages of the back office. |
+| Orders & Customers | Strings for the “Orders”, “Customers” and “Customer Service” sections from the back office. |
+| Catalog | Strings from the “Catalog” section. |
+| Modules | Strings from the Modules page. |
+| Design | Strings from the “Design” section. |
+| Shipping | Strings from the “Shipping” section. |
+| Payment (megashopba) | Strings from the “Payment” section. |
+| International | Strings form the “International” section. |
+| Parameters | Content from the “Shop Parameters” and “Advanced Settings” sections. |
+
+
+If you’re not familiar with the new sections in PrestaShop 1.7, [read this article about the new menu](http://build.prestashop.com/news/how-we-reorganized-main-menu-prestashop-1.7/).
+
+From “Orders & Customers” to “Parameters”, the third levels are organized the same way, as functional domains. Each of these third levels will include three sublevels:
+
+* **Feature**: everything specific to a functional domain. If you’re in the “Shipping” section, the domain *Admin.Shipping.Feature* will contain all the strings displaying in the Shipping menu, that are particular to these pages.
+* **Notification**: info, success, warning or error messages specific to the functional domain and the pages it includes.
+* **Help**: help text and placeholders.
+
+![Back office domains on Crowdin](/assets/images/2016/07/domains_admin.png)
+
+For example, here’s what it will give for a specific page from the back office (Design > Image settings)
+
+![Domains example for back office](/assets/images/2016/07/domains_example2.png)
+
+With the following domains being used:
+
+<img style="border: 1px solid #CCC; padding: 10px;" width="250" src="/assets/images/2016/06/domains_example3.png">
+
+
+
+## Limitations
+While we wanted to have a great system, there are nevertheless a few drawbacks.
+
+Where we used to have a string with a feature, and the following string the help text attached to it; they will now be in two different files.
```

Comment:
+Where we used to have a string with a feature, and in the following string the help text attached to it; they will now be in two different files.